src/Microsoft.ML.Data/Utils/LossFunctions.cs

```diff
@@ -424,13 +523,23 @@ internal sealed class PoissonLossFactory : ISupportRegressionLossFactory
     }
 
     /// <summary>
-    /// Poisson Loss.
+    /// Poisson Loss function for Poisson Regression.
     /// </summary>
+    /// <remarks type="text/markdown"><![CDATA[
+    ///
+    /// The Poisson Loss function is defined as:
+    ///
+    /// $L(f(\vec{x}), y) = e^{f(\vec{x})} - yf(\vec{x})$
+    ///
+    /// where $f(\vec{x})$ is the predicted value, $y$ is the true label.
```

Comment:
>$f(\vec{x})$ [](start = 14, length = 12)

For all the loss functions, I think it would be easier to read if this expression had a name, such as \hat(y). In the log loss case, you can define the probability p as a function of \hat(y), and then use p in the formula for the loss.